tech.json

```diff
@@ -495,7 +540,10 @@
           "extensionTechs": [
             {
               "name": "canJumpIntoIBJ",
-              "requires": [ "canIBJ" ],
+              "requires": [ 
+                "canMidAirMorph",
```

Comment:
or spring ball (this will break artificial morph strats that start with a spring ball)